EDIT/src/EDIT.py

```diff
@@ -0,0 +1,36 @@
+class EDIT():
+    def __init__(self,a,b):
+        self.a = a
+        self.b = b
+        self.len_a = len(a)
+        self.len_b = len(b)
+        self.edit_distance_dictionary = {}
+        self.answer = self.__get_edit_distance()
+
+    def __get_edit_distance(self):
+        self.__initialize_distances()
+        for i in range(1,self.len_a+1):
+            for j in range(1,self.len_b+1):
+                penalty = 1 if self.a[i - 1] != self.b[j - 1] else 0
+                self.edit_distance_dictionary[str(i)+':'+str(j)] = min(self.__get_distance(i - 1, j) + 1,
```

Comment:
extremely picky: since the logic of calculating the score is the key aspect of the algorithm, would recommend structuring the code so that it can be tested directly